catalog/spatial/geocoding/spatial-geocoding-feature/src/test/java/org/codice/ddf/spatial/geocoding/feature/TestCatalogFeatureQueryable.java

```diff
@@ -91,15 +100,41 @@ public void testQuery()
     assertThat(geometry.getGeometryType(), is(WKT_TYPE));
 
     QueryImpl query = (QueryImpl) requestArgument.getValue().getQuery();
-    String queryString = query.getFilter().toString();
 
+    Filter filter = query.getFilter();
+
+    And and = (And) filter;
+
+    List<Filter> filters = and.getChildren();
+    assertThat(filters, hasSize(3));
+
+    List<String> attributes = new ArrayList<>();
+    List<String> values = new ArrayList<>();
+    for (Filter compFilter : filters) {
+      String attribute;
+      String value;
+      if (compFilter instanceof IsEqualsToImpl) {
+        IsEqualsToImpl equals = (IsEqualsToImpl) compFilter;
+        AttributeExpressionImpl attributeExpression =
+            (AttributeExpressionImpl) equals.getExpression1();
+        attribute = attributeExpression.getPropertyName();
+        LiteralExpressionImpl literalExpression = (LiteralExpressionImpl) equals.getExpression2();
+        value = (String) literalExpression.getValue();
+      } else {
+        LikeFilterImpl likeFilter = (LikeFilterImpl) compFilter;
+        AttributeExpressionImpl literalExpression =
+            (AttributeExpressionImpl) likeFilter.getExpression();
+        attribute = literalExpression.getPropertyName();
+        value = likeFilter.getLiteral();
+      }
+      attributes.add(attribute);
+      values.add(value);
+    }
+
+    assertThat(attributes, hasItems(Location.COUNTRY_CODE, Core.METACARD_TAGS, Core.METACARD_TAGS));
     assertThat(
-        queryString,
-        containsString(String.format("metacard-tags is like %s", GeoCodingConstants.DEFAULT_TAG)));
-    assertThat(
-        queryString,
-        containsString(String.format("metacard-tags is like %s", GeoCodingConstants.COUNTRY_TAG)));
-    assertThat(queryString, containsString(String.format("title = %s", COUNTRY_CODE)));
+        values,
+        hasItems(COUNTRY_CODE, GeoCodingConstants.DEFAULT_TAG, GeoCodingConstants.COUNTRY_TAG));
```

Comment:
nit: it might also be useful to assert that the size of `attributes` and `values` is 3

Reply:
added